qa/tests/features/new/manual/dashboard/dashboard.feature

```diff
@@ -0,0 +1,120 @@
+@regression
+Feature: Filter refiner, Decision tree, Dashboard
+As an anfisa user I want to create custom layout for attributes
+
+Background: 
+	
+	Given Filter refiner or Decision tree of "xl_PGP3140_wgs_NIST-4_2" was opened
+
+Scenario: 01 Open dashboard
+	
+	When User clicks four-square shaped button near the attribute search panel
+	Then Dashboard should be opened
+	And All attributes should be presented
+
+Scenario: 02 Close section tab
+
```

Comment:
Did you miss "User clicks four-square shaped button near the attribute search panel"?
Indeed, this button name is too long. Anastasia says, that common name in industry for such icons is "card view"
So, better to use "User clicks "Card view" button"

Reply:
In this scenario and below user
1) Opens filter page for dataset 
2) Tries to click something on dashboard _without_ switching from list view to card view
Please, fix it.